src/components/navbar/Navbar.jsx

```diff
@@ -20,21 +24,40 @@ const navbarLinks = [
     name: "FAQs",
   },
 ];
-
 const Navbar = () => {
+  const [isNavOpen, setIsNavOpen] = useState(false);
+  const [width,setWidth]=useState(window.innerWidth);
```

Comment:
Do check if `window` object is null or not before using it inside `useState` hook. You can also use a ternary operator for initializing it with some predefined value, otherwise use `setState()` inside `useEffect` for assigning the value after `window` has been initialized.